### PR TITLE
Add Bazel instructions

### PR DESCRIPTION
**Pull request by ryanipete**

### CODE REVIEW

File: docs/ios/bazel.md [hunk 1]
```diff
@@ -0,0 +1,100 @@
+# iOS Build Instructions - Bazel
+
+## 1. Copy `productscience.yaml` to your project's root directory
+
+Product Science should provide you with a `productscience.yaml` file that contains your credentials and configuration details. If you never received this file or need to request a new copy, please contact support@productscience.ai.
```
Comment (ryanipete) [inline docs/ios/bazel.md:5]: Not sure what contact info to provide here. I guessed at "support@productscience.ai" but don't know if that's a valid email address.

**Comment by ryanipete** [comment]
I'm not sure what to do about the "example" section now that we automatically update the user's hotfix filters. Ideas?

**Comment by ryanipete** [comment]
Once we've settled on a final version of these instructions, I'll update the xcodebuild instructions to match.

File: docs/ios/bazel.md (hunk 1 above)
Comment (gmorgachev) [reply docs/ios/bazel.md:5]: Please ask @NateCham

File: docs/ios/bazel.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+Continuing with the previous example - As before, your project's root directory is `/code/MyApp`. 
+
+Assume that you use the command `bazel build //ios-app:ios-app` to build your project.
+ 
+Given the information above, your default `PSCliCodeInjector` command will be:
+
+```shell
+PSCliCodeInjector "/code/MyApp" \
+  --console-build-command="bazel build //ios-app:ios-app"
+```
```
Comment (gmorgachev) [inline docs/ios/bazel.md:68]: Do we have this MyApp in demoapps? To attach project on which client can check it?

File: docs/ios/bazel.md (hunk 1 above)
Comment (gmorgachev) [reply docs/ios/bazel.md:5]: Please contact Nate

File: docs/ios/bazel.md (hunk 2 above)
Comment (gmorgachev) [inline docs/ios/bazel.md:68]: Let's publish small app for quick testing

**Review by gmorgachev — APPROVED** [review]
1. I would add link to small app for quick test
2. Please add that page in `mkdocs.yml`

File: docs/ios/bazel.md (hunk 1 above)
Comment (NateCham) [reply docs/ios/bazel.md:5]: support@productscience.ai works, I would also add Slack "please contact your designated integration support via previously agreed upon email/shared Slack channel or support@productscience.ai if neither of those are available"

Comment (ryanipete) [reply docs/ios/bazel.md]: 👍  updated.

**Comment by ryanipete** [comment]
PR for Bazel version of SimpleApp is [here](https://github.com/product-science/demoapps/pull/9). Working now on updating these docs to reference that project.

**Comment by ryanipete** [comment]
Moving this PR to draft until internal code is updated to match documentation.

File: docs/ios/bazel.md [hunk 3]
```diff
@@ -0,0 +1,230 @@
+# iOS Build Instructions - Bazel
+
+The following instructions describe how to add Product Science instrumentation to a [Bazel](https://bazel.build/) project's iOS application.
+
+!!! example
```
Comment (ryanipete) [inline docs/ios/bazel.md:5]: image

File: docs/ios/bazel.md [hunk 4]
```diff
@@ -0,0 +1,230 @@
...
+    % git clone https://github.com/product-science/demoapps
+    ```
+
+    Our examples will assume that you've cloned the repository to your home directory and kept the repository's default directory name (`demoapps`). If you decide to change either of these details, then modify the specified paths to fit your setup.
+
+## 1. Copy `.productscience.yaml` {TODO-02} {TODO-08} to your home directory
+
+Product Science will provide you with a `.productscience.yaml` {TODO-02} file that contains your credentials and configuration details. 
+
+!!! info
```
Comment (ryanipete) [inline docs/ios/bazel.md:20]: image

File: docs/ios/bazel.md [hunk 5]
```diff
@@ -0,0 +1,230 @@
...
+
+Product Science will provide you with a `.productscience.yaml` {TODO-02} file that contains your credentials and configuration details. 
+
+!!! info 
+    
+    If you haven't received this file or need to request a new copy, please reach out to your designated integration support contact via email or Slack. If this contact is unavailable, send an email to support@productscience.ai for assistance.
+
+Once you've received your `.productscience.yaml` {TODO-02} file, copy the file to your home directory {TODO-03}.
+
+??? example
```
Comment (ryanipete) [inline docs/ios/bazel.md:26]: Collapsed:
image

Expanded:
image

The other `??? example` sections have the same appearance.

File: docs/stylesheets/extra.css [hunk 6]
```diff
@@ -162,11 +170,22 @@ header.md-header {
...
 /* .md-typeset code {
     font-size: .6.5rem;
-} */
\ No newline at end of file
+} */
+
+.md-typeset .admonition,
+.md-typeset .example {
+    font-size: .8rem;
+}
```
Comment (ryanipete) [inline docs/stylesheets/extra.css:186]: Increasing font size of "Info", "Example", "Warning" sections...

File: docs/stylesheets/extra.css [hunk 7]
```diff
@@ -162,11 +170,22 @@ header.md-header {
...
+
+.md-typeset .admonition .admonition-title,
+.md-typeset .example summary {
+    font-size: .6rem;
+}
```
Comment (ryanipete) [inline docs/stylesheets/extra.css:191]: ...but leaving their titles small.

**Pull request merged**